pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -373,6 +368,11 @@ func generateConfig(k8s config.KubernetesConfig) (string, error) {
 		NodeName:          k8s.NodeName,
 		ExtraArgs:         extraComponentConfig,
 		FeatureArgs:       kubeadmFeatureArgs,
+		NoTaintMaster:     false,
+	}
+
+	if version.GTE(semver.MustParse("1.10.0-alpha.0")) {
+		opts.NoTaintMaster = true
```

Comment:
This comparison breaks normal working of minikube with kubernetes older than 1.10.0-alpha.0, because taint is added and some addons do not tolerate it. Why we need NoTaintMaster at all?